docs/pages/introduction.mdx

```diff
@@ -1,5 +1,16 @@
 # What is MUD?
 
+<iframe
+  width="560"
+  height="315"
+  src="https://www.youtube.com/embed/Unfj6yN9wmA?si=0bfRVGFb2V_CDJ-z"
+  title="YouTube video player"
+  frameborder="0"
+  allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share"
+  referrerpolicy="strict-origin-when-cross-origin"
+  allowfullscreen
+></iframe>
```

Comment:
Can we make this look better? I feel like this disrupts the flow of the page

<img width="1225" alt="image" src="https://github.com/user-attachments/assets/d824d175-e295-42ce-8edc-4a50a716a449">